app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -360,6 +360,34 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
     border: 1px solid var(--base-10);
   }
 
+  .current-user-info {
+    text-align: center;
+    margin-bottom: 1rem;
+
+    .current-user-avatar-container {
+      height: 80px;
+    }
+
+    .current-user-avatar {
+      border-radius: 100%;
+      height: inherit;
+      object-fit: cover;
+      border: 2px solid var(--base-90);
+    }
+
+    h3 {
+      margin-top: 6px;
+    }
+
+    p {
+      font-size: 16px;
+      height: 24px;
+      color: var(--base-70);
+      margin-top: -10px
```

Comment:
Usually when I'm manipulating the spacing between text elements, it is not good practice to use negative values to bring them closer together. Doing that signals that there is a potential to do it better using line-height. I think that if you adjusted the line height of the `h3` and `p` element, and had their margin be `0` so that they touched, that would be more maintainable.

Reply:
Thanks for this! Updated!